01-Git_Bash_Make/Task2_bash_script/script

```diff
@@ -6,6 +6,8 @@ if ! [ -d $directory ]
 then
 	cp -R /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task1_game $directory
 	tar -cvzf $directory/Task1_game.tar.gz $directory
+	mkdir /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task2_bash_script/release
+	cp $directory/Task1_game.tar.gz /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task2_bash_script/release
```

Comment:
You're using the same long path several times (I'm leaving aside hardcodding such paths) - make sense to define a variable.

Reply:
It will be grate idea to define variable for temp and release dir